app/assets/javascripts/fae/form/_form.js

```diff
@@ -23,8 +23,35 @@ Fae.form = {
       $('.input.file').fileinputer();
     }
 
+    // Mutate DOM to support two column labels for all standard inputs
+    this.make_two_column_labels();
+
     // make all the hint areas
     $('.hint').hinter();
   },
 
+  make_two_column_labels: function() {
+    $('.input label').each(function() {
+      var $element = $(this);
+
+      // Bail if we cannot find any helper_text
+      if (!$element.find('.helper_text').length) { return; }
+
+      // If present, get all DOM nodes w/ contents(), but ignore the .helper_text
+      var label_inner = $element.contents().filter(function() {
+        return !$(this).hasClass('helper_text');
+      });
+      var helper_text = $element.find('.helper_text');
+
+      // Replace existing label w/ newly wrapped elements, sans .helper_text
+      label_inner = $('<div class="label_inner" />').html(label_inner);
```

Comment:
It may make more sense to use jQuery's `wrap()`. I'm not sure, but you may be able to consolidate the next line too using `wrap`, i.e. `$element.wrap( $('<div class="label_inner" />').wrap(label_inner) )`